Unconditional img generation - about.md file (#403)
huggingface/hub-docs: tasks/src/unconditional-image-generation/about.md

## About the Task

> Unconditional image generation is the task of generating images with no condition in any context (like a *prompt text* or *another image*). Once trained, the model will create images that resemble its training data distribution.

Reviewer: ```suggestion
Unconditional image generation is the task of generating images with no condition in any context (like a *prompt text* or *another image*). Once trained, the model will create images that resemble its training data distribution.
```
No need to write this given it's also on `data.ts` no?

Author: Oh sure! :)

> Unconditional image generation models usually start with a *seed* that generates a *random noise vector*. The model will then upsample this vector to create the final image. 

Reviewer: Can you explain upsampling a little 😅

> An example of unconditional image generation would be generating the image of a face on a model trained with the [CelebA dataset](https://huggingface.co/datasets/huggan/CelebA-HQ).

Reviewer: ```suggestion
An example of unconditional image generation would be generating an image of a face on a model trained with the [CelebA dataset](https://huggingface.co/datasets/huggan/CelebA-HQ).
```

> ## Use Cases

Reviewer: Can you leave these sections descriptions as well so it's easier for others to contribute🙂

## About the Task

Unconditional image generation models usually start with a *seed* that generates a *random noise vector*. The model will then use this vector to create an output image similar to the images used for training the model. 

An example of unconditional image generation would be generating the image of a face on a model trained with the [CelebA dataset](https://huggingface.co/datasets/huggan/CelebA-HQ) or [generating a butterfly](https://huggingface.co/spaces/huggan/butterfly-gan) on a model trained with the [Smithsonian Butterflies dataset](https://huggingface.co/datasets/ceyda/smithsonian_butterflies).

> [GAN](https://en.wikipedia.org/wiki/Generative_adversarial_network) and [Diffusion](https://huggingface.co/docs/diffusers/index) are common models for this task.

Reviewer: ```suggestion
[Generative adversarial networks](https://en.wikipedia.org/wiki/Generative_adversarial_network) and [Diffusion](https://huggingface.co/docs/diffusers/index) are common architectures for this task.
```
